### PR TITLE
Update function 3

### PR DESCRIPTION
**Pull request by PBaloyi449**

### CODE REVIEW

**Comment by PBaloyi449** [comment]
Updating the third function

File: index.js [hunk 1]
```diff
@@ -1,42 +1,29 @@
-let state = 'idle'
-let user = null
-let calculated = '1'
+let state = 'idle';
```
Comment (Lesego-Shange) [inline index.js:1]: No changing above line

File: index.js [hunk 2]
```diff
@@ -1,42 +1,29 @@
...
-    const isString = typeof calculated = 'numerical-string' 
-    const calculatedAsNumber = isString ? calculated : parseNumber(calculated)
-    calculated === calculatedAsNumber + 1 
-}
-
-const = calcUser () => {
-  logCalc
-  if (calculated > 2) user = 'John'
-  if (calculated > 2) state = 'requesting'
-  if (calculated > 3) state = 'idle'
```
Comment (Lesego-Shange) [inline index.js:18]: Do not delete other code

File: index.js [hunk 3]
```diff
@@ -1,42 +1,29 @@
...
+const checkUser = () => {
```
Comment (Lesego-Shange) [inline index.js:19]: Please perform fetch then retry the commit

**Review by Lesego-Shange — CHANGES_REQUESTED** [review]
Please make the changes as requested then resubmit

**Pull request closed without merging**